sw/device/silicon_creator/rom/doc/rom_overview_specification.md

```diff
@@ -0,0 +1,272 @@
+# Mask ROM Specification
+- Authors: cfrantz
+- Reviewers: alphan, moidx
+- Date: 2021-06-28
+- Status: Implmented
+
+
+## Overview
+
+This document is a high-level overview of the features and responsibilities of the Mask ROM.
+
+This document is not a comprehensive specification.  When appropriate, it refers to more complete specifications for various mask ROM responsibilities.  These specifications include:
+
+
+
+*   [Signature Verification Module [sigverify]](#sigverify)
```

Comment:
nit: should this list use regular parentheses instead of square brackets for short names?

Reply:
I'd rather use square brackets.  Parentheticals are for providing additional information or side-commentary.

The square brackets are used because the words are meant to be read and to signal a reference to an external spec.